### ISSUE TITLE
Multi Filters in a feed

### ISSUE THREAD

**Issue by sean89503** [mxpv/podsync#158]
Its not clear if this is possible or how to do Multiple filters on a feed, example: I want to filer episodes that are "Live" or if a episode has "Q&A"

so would i use 
not_title = "Live" OR "LIVE" OR "Q&A" OR "q&a"

**Comment by Rpsl** [comment]
Yep. You can use regular expression for combination of multiple filters

- example: https://regex101.com/r/4qaQT2/2
- docs: https://golang.org/pkg/regexp/syntax/

**Comment by mxpv** [comment]
Should probably update the documentation with more examples on this.

**Comment by tapatianbeast** [comment]
> Yep. You can use regular expression for combination of multiple filters
> 
> * example: https://regex101.com/r/4qaQT2/2
> * docs: https://golang.org/pkg/regexp/syntax/

How i can get the sentence “full episode" in a string?

**Comment by bw4518** [comment]
What if I want to combine title and min_duration filter? I want to get everything matching the title OR min_duration. Is that possible?

**Comment by mxpv** [comment]
@claude can you update documentation and provide a few examples? Be sure to write unit tests for filter examples you introduce (similar to https://github.com/mxpv/podsync/commit/5c734999fea4b6eb2b05d6b07fd7fee9ba976803).
